libs/dh/metering-point/feature-overview/src/lib/primary-master-data/dh-metering-point-primary-master-data.component.html

```diff
@@ -0,0 +1,105 @@
+<!--
+@license
+Copyright 2021 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<ng-container
+  *transloco="let transloco; read: 'meteringPoint.overview.primaryMasterData'"
+>
+  <section>
```

Comment:
question: Have you tried wrapping the sections in a flexbox and control them that way? The actual address part is in a funny position on smaller screens. Shouldn't it be closer to the address at the top? With flexbox it will be easier to have these two together.

![image](https://user-images.githubusercontent.com/1096332/146381332-69bf2e1d-b428-40b4-8631-f2ae782a18d3.png)